### PR TITLE
purge all the -v references from e2e.go

### PR DESCRIPTION
**Pull request by krzyzacy**
cf for https://github.com/kubernetes/kubernetes/issues/60870

probably need to work together with https://github.com/kubernetes/test-infra/pull/7164

### CODE REVIEW

**Comment by krzyzacy** [comment]
okay, kubemark is green now, someone give me a stamp?

File: test/kubemark/run-e2e-tests.sh [hunk 1]
```diff
@@ -44,7 +44,7 @@ fi
 
 if [[ -f /.dockerenv ]]; then
 	# Running inside a dockerized runner.
-	go run ./hack/e2e.go -- -v --check-version-skew=false --test --test_args="--e2e-verify-service-account=false --dump-logs-on-failure=false ${ARGS}"
+	go run ./hack/e2e.go -- --check-version-skew=false --test --test_args="--e2e-verify-service-account=false --dump-logs-on-failure=false ${ARGS}"
```
Comment (krzyzacy) [inline test/kubemark/run-e2e-tests.sh]: @shyamjvs I think this command need to live inside kubetest, I feel like the root issue is kubetest used in e2e.go is not the same with kubetest in the kubekins image

**Review by BenTheElder — APPROVED** [review]
/lgtm

**Comment by ixdy** [comment]
in case you're wondering like I was, https://github.com/kubernetes/test-infra/pull/7073/commits/f85613d61242179bfc7804ce64d93182cad1f35b#r172374565 explains that verbose mode is now default, hence no need for `-v` (which was also removed/renamed to `-verbose`).

**Comment by shyamjvs** [comment]
Thanks for fixing Sen! Btw - I realized what the issue was as soon as I
read the subject line, because I faced the problem with -v flag just
yesterday :)

**Comment by BenTheElder** [comment]
https://kubernetes.slack.com/archives/C09QZ4DQB/p1520441116000310

We need to advertise this well, lots of people have `go run hack/e2e.go -v ...` in their shells etc.

**Comment by krzyzacy** [comment]
yeah, I was kinda expecting `-v` will still kinda work, since that's the flag for glog now, did not realize that -v need to take in a log tier as well :-\

@shyamjvs how do you feel about move this script, at least the docker part, into kubetest, to avoid similar breakage in the future?

**Comment by shyamjvs** [comment]
Needs more thinking - but I believe it should be possible. How urgent is
it? (I'm having ~0 free cycles atm:)
Also, if someone is willing to do it, I can provide some input.

**Comment by krzyzacy** [comment]
@shyamjvs not really since the issue is fixed - we can wait after the release, and I can also help on that.

**Comment by krzyzacy** [comment]
@ixdy what if people want to use it for glog? cc @Q-Lee

**Comment by BenTheElder** [comment]
https://github.com/kubernetes/kubernetes/pull/60895 for translating to the
new flag. if users want to control glog they should use kubetest directly.

**Comment by krzyzacy** [comment]
1.9 one is in, waiting for 1.8 and 1.7

/milestone v1.8

**Pull request merged**